actors/paych/src/lib.rs

```diff
@@ -53,16 +52,10 @@ impl Actor {
         // behalf of the payer/payee.
         rt.validate_immediate_caller_type(std::iter::once(&Type::Init))?;
 
-        // Resolve both parties, confirming they exist in the state tree.
-        let to = Self::resolve_address(rt, &params.to)
-            .with_context_code(ExitCode::USR_ILLEGAL_ARGUMENT, || {
-                format!("to address not found {}", params.to)
-            })?;
+        // Check both parties are capable of signing vouchers
+        let to = Self::resolve_account(rt, &params.to)?;
```

Comment:
Just call `resolve_to_actor_id(rt, params.to, true`). Please delete the `resolve_account` method.